presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/runneraction/RunnerAction.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.runneraction;
+
+import com.facebook.presto.jdbc.QueryStats;
+import com.facebook.presto.sql.tree.Statement;
+import com.facebook.presto.verifier.framework.QueryResult;
+import com.google.common.collect.ImmutableList;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public interface RunnerAction
```

Comment:
This should be called `PrestoAction`. It means performing an action against Presto.